internal/auth0/branding/resource.go

```diff
@@ -87,10 +95,10 @@ func NewResource() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"body": {
-							Type:        schema.TypeString,
-							Optional:    true,
-							Computed:    true,
-							Description: "The body of login pages.",
+							Type:         schema.TypeString,
+							Required:     true,
```

Comment:
Marking this as required to prevent instances where the `font` block is present with no attributes assigned.